corehq/apps/app_manager/models.py

```diff
@@ -718,6 +718,8 @@ def __get__(self, form, form_cls):
             source = ''
         else:
             source = app.lazy_fetch_attachment(filename)
+            if isinstance(source, bytes):
```

Comment:
This should always be bytes now, right? So the conditional is unnecessary.

Reply:
That's what I thought ... but in tests ```__set__``` is used with unicode, which then causes ```lazy_fetch_attachment``` to return unicode.  I plan on cleaning up these tests, but these simple changes should resolve the P3 quickly.

Reply:
More specifically, I think it's reasonable to have ```source``` always get and set unicode, but we need to encode before putting the attachment.  That seems like the right boundaries for the unicode sandwich.